tests/test_on_data/test_sorting_interfaces.py

```diff
@@ -2,7 +2,9 @@
 from unittest import TestCase
 
 import numpy as np
+from numpy.testing import assert_array_equal
 from pynwb import NWBHDF5IO
+from spikeinterface.extractors.nwbextractors import read_nwbfile
```

Comment:
I am using `read_nwbfile` because I want to read the data in to `NWBFile` object and confirm that the `max_channel` column was properly written to the units table. This seemed like a convenient way to read the data but I agree it's a bit strange to rely on SpikeInterface for this function.

Reply:
Ah, Ben, my confusion was worse than that. Sadly, we have that function name duplicated in that space and I thought you were calling this function (which is at the bottom of the same file):


https://github.com/h-mayorquin/spikeinterface/blob/2085cbe9a7fb74ce5d9c2c831611eb2a596de686/src/spikeinterface/extractors/nwbextractors.py#L1367-L1368